hack/install-ocs-operator.sh

```diff
@@ -33,28 +33,39 @@ data:
 EOF
 
 
-# Ensure position independent make targets in release CI, explicitly setting the values ensures client-op doesn't deploy CSI
-# when storagecluster is configured for remoteconsumers, controllers set this value to "true"
-cat <<EOF | oc create -f -
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: ocs-client-operator-config
-  namespace: openshift-storage
-data:
-  DEPLOY_CSI: "false"
-EOF
+patch_ocs_client_operator_config_configmap() {
+    while true; do
+        if oc get cm ocs-client-operator-config -n openshift-storage; then
+            oc patch cm ocs-client-operator-config -n openshift-storage --type merge -p '{"data":{"DEPLOY_CSI":"false"}}'
+            sleep 10
```

Comment:
is any external entity resetting it? if yes, it can do so after sleep as well, if the api returned successfully after a patch then we could directly test the value isn't it?

Reply:
If we directly check the value without sleep, we may get the value that we set, and the external entity has not reset it yet. I added this after testing only.

Reply:
May I know what made you pick `10s`? I don't think sleep is a good protection against external entity and is csv-controller the one changing fields? Did you check managed-fields on cm before coming up w/ sleep?

I'm just being extra cautious at the expense of delaying the merge as this is a high frequency script, pls excuse.

Reply:
I believe it is CSV controller but I am not sure. I used 10 second just like that and tested it multiple times and it worked.

Reply:
> multiple times and it worked.
- ack, basing my vote only on this.